air-quality-backend/tests/api/mappers/forecast_mapper_test_data.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
+
+from bson import ObjectId
+
+forecast_from_database_for_mapper = [
+    {
+        "_id": ObjectId("66558bf12d46a42baea0b8e1"),
+        "forecast_base_time": datetime.strptime(
+            "2024-05-27T12:00:00+00:00", "%Y-%m-%dT%H:%M:%S%z"
+        ),
+        "forecast_valid_time": datetime.strptime(
+            "2024-05-27T12:00:00+00:00", "%Y-%m-%dT%H:%M:%S%z"
+        ),
+        "name": "Abidjan",
+        "source": "cams-production",
+        "location_type": "city",
+        "last_modified_time": datetime.strptime(
+            "2024-05-27T12:00:00+00:00", "%Y-%m-%dT%H:%M:%S%z"
+        ),
+        "created_time": datetime.strptime(
+            "2024-05-27T12:00:00+00:00", "%Y-%m-%dT%H:%M:%S%z"
+        ),
+        "location": {"type": "Point", "coordinates": [-4.01266, 5.30966]},
+        "forecast_range": 0,
+        "overall_aqi_level": 2,
+        "o3": {"aqi_level": 1, "value": 48.8483987731408},
+        "no2": {"aqi_level": 1, "value": 0.3145229730198031},
+        "so2": {"aqi_level": 1, "value": 0.676714188255428},
+        "pm10": {"aqi_level": 2, "value": 24.464592631770792},
+        "pm2_5": {"aqi_level": 2, "value": 14.396278071945583},
+    },
+    {
+        "_id": ObjectId("66558bf12d46a42baea0b8e4"),
+        "forecast_base_time": datetime.strptime(
+            "2024-05-27T13:00:00+00:00", "%Y-%m-%dT%H:%M:%S%z"
+        ),
+        "forecast_valid_time": datetime.strptime(
+            "2024-05-27T13:00:00+00:00", "%Y-%m-%dT%H:%M:%S%z"
+        ),
+        "name": "Abidjan",
+        "source": "cams-production",
+        "location_type": "city",
+        "last_modified_time": datetime.strptime(
+            "2024-05-27T12:00:00+00:00", "%Y-%m-%dT%H:%M:%S%z"
+        ),
+        "created_time": datetime.strptime(
+            "2024-05-27T12:00:00+00:00", "%Y-%m-%dT%H:%M:%S%z"
+        ),
+        "location": {"type": "Point", "coordinates": [-4.01266, 5.30966]},
+        "forecast_range": 0,
+        "overall_aqi_level": 2,
+        "o3": {"aqi_level": 1, "value": 48.8483987731408},
+        "no2": {"aqi_level": 1, "value": 0.3145229730198031},
+        "so2": {"aqi_level": 1, "value": 0.676714188255428},
+        "pm10": {"aqi_level": 2, "value": 24.464592631770792},
+        "pm2_5": {"aqi_level": 2, "value": 14.396278071945583},
+    },
```

Comment:
At a glance these two objects look the same and have the same values. 

Why don't we make a function that creates an object with all of these fields and as a parameter it takes in a dictionary. The dictionary can then be used to override the default values. So something like this:

```python
def create_mock_forecast_document(overrides):
  default_document = {
    # put the defaults here
  }
  return {
    **default_document,
    **overrides
  }
```

Then we can call this from within the test rather than have everything in separate places, so the test should be easier to understand just by looking at it and if we override something we can see what is important in the test